client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -45,16 +53,17 @@ export default function UserOperatorMultiStepForm({
     setFormState(newFormData);
 
     // add user operator id to form data if it exists (to be used in senior officer creation)
-    if (userOperatorId) newFormData.user_operator_id = userOperatorId;
+    if (userOperatorId)
+      newFormData.user_operator_id = userOperatorIdParam || userOperatorId;
 
     const apiUrl = `registration/user-operator/${
       isFinalStep ? "contact" : "operator"
     }`;
 
     const response = await actionHandler(
-      apiUrl,
-      "POST",
-      `/dashboard/select-operator/user-operator/create/${params?.formSection}`,
+      `${apiUrl}${!isCreate && !isFinalStep ? `/${userOperatorId}` : ""}`,
```

Comment:
@BCerki I've reverted this logic back to `!isCreate && !isFinalStep` as that is what broke the editing of page 2 working. If you can suggest another way to write it I will update it thanks.